src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryMath.cs

```diff
@@ -193,25 +331,88 @@ public static FormulaValue SumTable(EvalVisitor runner, SymbolContext symbolCont
         // Max(1,2,3)     
         internal static FormulaValue Max(IRContext irContext, FormulaValue[] args)
         {
-            return RunAggregator(new MaxAgg(), irContext, args);
+            var agg = GetMinMaxAggType(irContext, false);
+
+            if (agg != null)
+            {
+                return RunAggregator(agg, irContext, args);
+            }
+            else
+            {
+                return new BlankValue(irContext);
+            }
         }
 
         // Max([1,2,3], Value * Value)     
         public static FormulaValue MaxTable(EvalVisitor runner, SymbolContext symbolContext, IRContext irContext, FormulaValue[] args)
         {
-            return RunAggregator(new MaxAgg(), runner, symbolContext, irContext, args);
+            var agg = GetMinMaxAggType(irContext, false);
+
+            if (agg != null)
+            {
+                return RunAggregator(agg, runner, symbolContext, irContext, args);
+            }
+            else
+            {
+                return new BlankValue(irContext);
+            }
         }
 
         // Min(1,2,3)     
         internal static FormulaValue Min(IRContext irContext, FormulaValue[] args)
         {
-            return RunAggregator(new MinAgg(), irContext, args);
+            var agg = GetMinMaxAggType(irContext, true);
+
+            if (agg != null)
+            {
+                return RunAggregator(agg, irContext, args);
+            }
+            else
+            {
+                return new BlankValue(irContext);
```

Comment:
When can this happen? Should it be an unreachable error?

Reply:
This would happen if irContext.ResultType wasn't a type we expected. But I don't think it should ever actually happen.

Reply:
so, in those cases, why return blank instead of an unreachablecode error?

Reply:
return CommonErrors.UnreachableCodeError(irContext);

Reply:
Added